python/README.md

```diff
@@ -0,0 +1,20 @@
+# Build Instructions
+
+## REST API
+
+There is currently a dependency to build from source in order to use the REST API. 
```

Comment:
Could we have a href for “REST API” for people who don’t know?

Reply:
Addressed in https://github.com/mlc-ai/mlc-llm/pull/223/commits/17277a394e7aeca54c2dbc6faef706ab1b8c201f.